src/main/java/lol/ui/StartMenu.java

```diff
@@ -0,0 +1,69 @@
+package lol.ui;
+
+import javafx.application.Application;
+
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.TilePane;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.shape.Rectangle;
+import javafx.scene.paint.Color;
+import javafx.scene.layout.*;
+import javafx.scene.text.*;
+
+import javafx.stage.Stage;
+import javafx.application.Platform;
+import javafx.geometry.Pos;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+
+import lol.game.*;
+import lol.server.*;
+import lol.ui.*;
+
+public class StartMenu extends Application {
```

Comment:
I think you can keep the same name (LOL2D) for this class. Otherwise it'll be painful to merge for everybody.